package.json

```diff
@@ -159,5 +159,9 @@
     "eslint-plugin-jsx-a11y": "^6.3.1",
     "eslint-plugin-react": "^7.20.5",
     "eslint-plugin-react-hooks": "^1.7.0"
+  },
+  "engines": {
+    "node": "15.13.0",
```

Comment:
Is there a way you can remove the restriction on the minor version? Is it okay?

Reply:
Done